WalletWasabi.Fluent/ViewModels/Wallets/Home/Tiles/PrivacyRing/PrivacyBarViewModel.cs

```diff
@@ -147,7 +146,6 @@ private IEnumerable<PrivacyBarItemViewModel> CreatePocketSegments(IEnumerable<Po
 			if (width < 2)
 			{
 				width++;
-				margin--;
```

Comment:
nACK: if you increase the width but don't reduce the margin you will end up filling more space than expected per segment, and thus filling the bar before you have chance to draw all segments. Unless I'm missing something here

Reply:
Ahh, you are right. 
Margins shouldn't be modified because the gaps between the segments must be equal in order to get an aesthetically good look.

Thanks for pointing it out, I will take a look.

Reply:
Pushed fixed, https://github.com/zkSNACKs/WalletWasabi/pull/9317/commits/ad5a338b4a03c6e7476e58ab999fa5394c75832f.

Although I didn't want to touch this drastically the code but didn't find a better solution as the main problem was that I had to correct all the other segments once a smaller one is corrected. (previously it was solved by changing the gaps, but that is what I wanted to avoid)
I put comments there to help to understand the logic.